product_docs/docs/mysql_data_adapter/2/02_requirements_overview.mdx

```diff
@@ -16,35 +18,15 @@ This table lists the latest MySQL Foreign Data Wrapper versions and their suppor
 | 2.5.5                      | N       | N       | Y       | Y       | Y       |
 | 2.5.4                      | N       | N       | Y       | Y       | Y       |
 
-## Supported MySQL versions
+## MySQL support
 
-The MySQL Foreign Data Wrapper is supported for MySQL 8 on the following platforms:
+MySQL Foreign Data Wrapper supports MySQL 8 on all supported MySQL Foreign Data Wrapper platforms. 
```

Comment:
MySQL Foreign Data Wrapper is repeated in this sentence. I am not sure, but can we try something else?